OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -390,7 +390,7 @@ public static void LoadActors(IniFile file, string section, List<string> players
 					if (health != 100)
 						initDict.Add(new HealthInit(health));
 					if (facing != 0)
-						initDict.Add(new FacingInit(facing));
+						initDict.Add(new FacingInit(255 - facing));
```

Comment:
I can see you're new here... there's no reason to configure this, because this code only deals with Westwood Tiberian Dawn/Red Alert maps which had only one way to define facings.

8bit int mix looks nicer though, but the hardcoding is absolutely justified.